### ISSUE TITLE
Setting browser:true in .jshintrc does not cover some globals

### ISSUE THREAD

**Issue by rbruckheimer** [jshint/jshint#3495]
The globals "customElements" and "ResizeObserver" ought to be added to the exports.browser object in vars.js.

**Comment by semmel** [comment]
Yes indeed, I need to tend an ever growing list of globals in my .jshintrc file:
```
"predef": [ "AbortController", "crypto", "DOMException", "NetworkInformation", "customElements", "URLSearchParams"]
```
It's a little sad. 😢

**Comment by jugglinmike** [comment]
Sounds about right to me. This is a [good first issue](https://github.com/jshint/jshint/labels/good%20first%20issue)--@rbruckheimer has explained exactly what needs to be done.

**Comment by JuanFML** [comment]
Hi I am new here I could add try to do this as my first PR, but do I only have to add "customElements" and "ResizeObserver", to  the exports.browser object in vars.js and set them as false? Or is something else needed first?

**Comment by semmel** [comment]
@JuanFML please add all [DOM interfaces](https://html.spec.whatwg.org/multipage/indices.html#all-interfaces) and [Window globals](https://developer.mozilla.org/en-US/docs/Web/API/Window)

That would be great!

**Comment by tusharkrbarman** [comment]
@jugglinmike I am new to Open source, Please assign this issue to me
